### PR TITLE
Accept `MANY` as the cardinality of license.

### PR DESCRIPTION
**Pull request by marcenacp**

### CODE REVIEW

**Review by ccl-core — APPROVED** [review]
Nice!

File: python/mlcroissant/mlcroissant/_src/core/json_ld.py [hunk 1]
```diff
@@ -102,6 +102,18 @@ def remove_empty_values(d: Json) -> Json:
     return {k: v for k, v in d.items() if v}
 
 
+def first_element_or_list(d: Any):
```
[the remark this answers is not in the document]
Comment (marcenacp) [reply python/mlcroissant/mlcroissant/_src/core/json_ld.py:105]: I agree the naming is not perfect... But I think it should be in a dedicated function as we use this logic a lot.

```
if cardinality == MANY:
  if LENGTH == 1:
    return array[0]
  else:
    return array
```

[the remark this answers is not in the document]
Comment (benjelloun) [reply python/mlcroissant/mlcroissant/_src/core/json_ld.py:105]: maybe_unbox_list ? or unbox_singleton_list ?

You could also simplify by making the function only work on lists, and check that the input is a list before calling it.

[the remark this answers is not in the document]
Comment (marcenacp) [reply python/mlcroissant/mlcroissant/_src/core/json_ld.py:105]: > You could also simplify by making the function only work on lists, and check that the input is a list before calling it.

The function should work on `Any`

**Pull request merged**